engine-rest/engine-rest/src/test/java/org/camunda/bpm/engine/rest/util/container/TomcatServerBootstrap.java

```diff
@@ -69,7 +69,6 @@ public void start() {
       .useLegacyLocalRepo(true).configureViaPlugin();
 
     WebArchive wa = ShrinkWrap.create(WebArchive.class, "rest-test.war").setWebXML(webXmlPath)
-        .addAsLibraries(resolver.resolve("org.codehaus.jackson:jackson-jaxrs:1.6.5").withTransitivity().asFile())
```

Comment:
@tmetzke shouldn't we replace this library with the `2.12.1` instead of removing it?

Reply:
That artifact does not exist anymore in version 2.x, the classes are provided with `com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider` which we already include with the dependency in line 39 of that same file 😃

Reply:
Not sure though if we would need to add this explicitly to the Tomcat deployment here but Shrinkwrap seems to be doing this already, otherwise the related tests would fail, as far as I understood this.

Reply:
Thanks for the explanation. In that case, it looks good.